src/app/services/login/login.service.ts

```diff
@@ -0,0 +1,50 @@
+import { Injectable } from '@angular/core';
+import { TranslateService } from '@ngx-translate/core';
+import { AccountService } from '../auth/account.service';
+import { AuthServerProvider } from '../auth/auth-jwt.service';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class LoginService {
+
+  constructor(
+    private accountService: AccountService,
+    private authServerProvider: AuthServerProvider,
+    private translate: TranslateService
+  ) {}
+
+  login(credentials, callback?) {
+      const cb = callback || function() {};
+
+      return new Promise((resolve, reject) => {
+          this.authServerProvider.login(credentials).subscribe(
+              data => {
+                  this.accountService.identity(true).then(account => {
+                      // After the login the language will be changed to
+                      // the language selected by the user during his registration
+                      // if (account !== null) {
+                      //     this.translate.use(account.langKey);
+                      // }
```

Comment:
Does this comment need to be removed, or this still needs to be implemented?

Reply:
These lines have been uncommented as they are useful for translations